Makefile

```diff
@@ -12,5 +35,19 @@ updaterdk:
 	go get go.viam.com/rdk@latest
 	go mod tidy
 
+FFmpeg:
+	git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
+
+$(FFMPEG_PREFIX): FFmpeg
```

Comment:
This will prevent slowdowns from configure calls if install path already exists.